share/spice/untappd/footer.handlebars

```diff
@@ -0,0 +1,7 @@
+<div class="untappd__footer tile__update__info two-line">
+    <span>{{abv}}% ABV</span>
+    <i class="sep"></i>
+    <span>{{ibu}} IBU</span>
+    <i class="sep"></i>
+    <span><i class="count__i"></i> {{ratings}}</span>
```

Comment:
> There are no ratings in untappd search results for now. We can find out ratings using beer info api but it will produce too many requests per search query.

@urbanserj so you're saying we could get them with an additional API call?

That's totally feasible/acceptable for Spices. The design team pointed this out as one of the things they felt should be improved with the IA, so if it's easily accessible I think we should show the rating because that's more important to the user I suspect?

The way it would work is that you would add an `alt_to` endpoint in the Perl, and then using the `onItemShown()` you would use `$.getJSON('/js/spice/untappd_rating/$id', function(data){ item.set({ rating: data.rating });` to update the `item` model. Along with out `rerender` property this would force the template to be updated accordingly.